content/en/docs/started/introduction.md

```diff
@@ -4,11 +4,43 @@ description = "An introduction to Kubeflow"
 weight = 1
 +++
 
-The Kubeflow project is dedicated to making deployments of machine learning (ML)
-workflows on Kubernetes simple, portable and scalable. Our goal is not to
-recreate other services, but to provide a straightforward way to deploy
-best-of-breed open-source systems for ML to diverse infrastructures. Anywhere
-you are running Kubernetes, you should be able to run Kubeflow.
+## What is Kubeflow ?
+
+Kubeflow is a community and ecosystem of open-source projects to address each stage in the
+machine learning (ML) lifecycle. It makes ML on Kubernetes simple, portable, and scalable.
+Its goal is to facilitate the orchestration of Kubernetes workloads for ML and to empower users to
+deploy best-in-class open-source systems for ML to diverse cloud infrastructures.
+Whether you’re a researcher, data scientist, ML engineer, or a team of developers, Kubeflow offers
+modular and scalable tools that cater to all aspects of the ML lifecycle: from building ML models to
+deploying them to production for AI applications.
+
+## What are Kubeflow Standalone Components?
+
+Kubeflow is composed of multiple, independent open-source projects which address different aspects
+of a ML lifecycle. These standalone components are designed to be usable both within the Kubeflow
+Platform and independently. These components can be installed on their own on a Kubernetes cluster,
+providing flexibility to users who may not require the full capabilities of Kubeflow Platform but
+wish to leverage specific functionalities in the ML lifecycle.
```

Comment:
```suggestion
wish to leverage specific functionalities.
```

Reply:
@StefanoFioravanzo What do you think about this ?
"""
but wish to leverage specific ML functionalities.
"""